src/test/java/com/exasol/adapter/dialects/ScalarFunctionsParameterCache.java

```diff
@@ -0,0 +1,62 @@
+package com.exasol.adapter.dialects;
+
+import java.io.*;
+import java.util.*;
+
+import org.yaml.snakeyaml.Yaml;
+
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * This class caches the parameters for runs that were passed the tests. By that in upcoming test runs can only test
+ * these runs and by that are a lot faster.
+ */
+public class ScalarFunctionsParameterCache {
+    private static final String CACHE_FILE_NAME = "src/test/resources/scalarFunctionsParameterCache.yml";
```

Comment:
Please create a folder in resources for the file. "integration" or something like this